src/guide/troubleshooting/configuration-issues.md

```diff
@@ -8,6 +8,15 @@ common source of issues in Iroha.
 If the issue you are experiencing is not described here, contact us via
 [Telegram](https://t.me/hyperledgeriroha).
 
-<!-- TODO: add -->
+## Fixing a `genesis` file issue on a Docker-compose setup
 
-**TBD**
+If you are using the Docker-compose version of Iroha and you have pulled the Iroha code the first time long ago, you may be surprised with Iroha working incorrectly. A mismatch between the versions of Iroha may lead to one of the peer containers failing with an error, related to the genesis file, i.e. a file that is needed to initialize the Iroha peer.
+
+To find out if this situation is applicable, you need to start by check the current containers using the `docker ps` command.
+Depending on the version, you may see either `hyperledger/iroha2:dev` or `hyperledger/iroha2:lts` containers. Pay attention to the number of the Iroha peer containers you are seeing in the `docker ps` output. By default, there are 4 peers configured in `docker-compose.yml` for Iroha, although you may have changed that value. Now, you may see that the first container that should have been running Iroha just exited with an error, while three other containers remain active.
+
+You can get a better understanding of this situation by checking the logs. If you've started your Iroha in daemon mode with `docker compose up -d`, you should refer to the `docker compose logs` command. You may also be running it directly with the `docker compose up` command. So, to check if you have this issue, look for an error text, containing "Failed to deserialize raw genesis block".
+
+The way to troubleshoot such an issue depends on the use of Iroha. It may be a basic demo, in which case the data isn't important and you can simply reset the genesis file to its latest state by typing `git checkout configs/peer/genesis.json`. A more complex situation with restoring an Iroha instance that was previously used will be documented later.
```

Comment:
`later` -- in this PR?

Reply:
We don't have some features like a genesis migration tool yet, but I've updated the text so it's more elaborate